ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -231,7 +314,14 @@ This was a lot of material, but you should have a healthy appreciation for the b
 
 #### Advanced querying
 
-1. Read section 14 in the [Rails Guide on Querying](https://guides.rubyonrails.org/active_record_querying.html#scopes) for a look at scopes. Again, you don't necessarily need to memorize all the details of scopes, but you should understand the concept and when it might be useful.
+1. Read section 14 in the [Rails Guide on
+   Querying](https://guides.rubyonrails.org/active_record_querying.html#scopes)
+   for a look at scopes. Again, you don't necessarily need to memorize all the
+   details of scopes, but you should understand the concept and when it might be
+   useful.
```

Comment:
```suggestion
1. Read section 14 in the [Rails Guide on
   Querying](https://guides.rubyonrails.org/active_record_querying.html#scopes)
   for a look at scopes. Again, you don't necessarily need to memorize all the
   details of scopes, but you should understand the concept and when it might be
   useful.
```
```suggestion
1. Read section 14 in the [Rails Guide on Querying](https://guides.rubyonrails.org/active_record_querying.html#scopes) for a look at scopes. Again, you don't necessarily need to memorize all the details of scopes, but you should understand the concept and when it might be useful.
```
Removing line breaks.